docs/manage/tpl/account/team.html

```diff
@@ -49,7 +49,10 @@ <h4>Account invitations</h4>
             <input type="hidden" name="a" value="[% account.id ? account.id_token : 'new' %]" />
             <input type="hidden" name="auth_token" value="[% combust.auth_token %]" />
             <div class="form-group">
-                <label id="name-invite_email" class="form-label" for="invite_email">Add user to account</label>
+                <label id="name-invite_email" class="form-label" for="invite_email">Add user to account team</label>
+                <br>
+                Note that all users have full access to the account
+                <p>
```

Comment:
I think this needs to be a <p></p> block; at least that's what the rest of the markup is doing.

Did you test that the note get indented correctly with the labels?

Reply:
Hi Ask, yes - I opened the current page in Chrome, opened up the Inspect pane, made the changes and the Note is indented to the same place as the "Add user..." heading, then copied out the changes to the source file.  The original note on line 69 isn't showing as a block.